packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -337,6 +344,8 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
         // state. As requirements change, so should these checks.
         assert(this.messageBuffer.length === 0, 0x0e9 /* "messageBuffer is not empty on new connection" */);
 
+        this.opsSize = 0;
```

Comment:
If this resets at connect, If I understand correctly, this may report a lot more op sizes, right? Basically everything in-between connects. If we end up in a disconnect loop, the first trace will probably be bad

Reply:
that is actually a good point - I wonder if I should also reset when we flush

Reply:
Tough question :) Was looking through the code and you're right, we may lose the size.

Reply:
I'm not sure what's the concern. There should be no ops sent while client is disconnected (this is enforced). And same ops maybe attempted to be sent on multiple connections - DM has no way to differentiate. I believe that's totally fine, if someone wants to capture how many unique ops were actually sent, they will need to add different type of telemetry (measure acked ops).

Reply:
That makes sense.

Reply:
My concern was as following:

- delta manager gets created, opsSize is 0
- successful payloads are being transmitted and opsSize is incremented with their lenghts.
- one large 1MB payload comes along, opSize += 1 MB
- disconnect happens, connectionProps will be logged, with opSize == (whatever was before the large payload + the large payload)
- opSize is 0 again
- the large payload is sent again, opSize == 1 MB
- disconnect happens, connectionProps will be logged, with opSize == the actual large payload

Not a big deal, but just saying that the first log statement will contain the large payload size + whatever was sent successfully before that. The subsequent log statements will be correct..